pystapi-client/pyproject.toml

```
@@ -16,10 +16,11 @@ dependencies = [
     "httpx>=0.28.1",
     "stapi-pydantic",
     "python-dateutil>=2.8.2",
+    "click>=8.1.8",
 ]
 
 [project.scripts]
-stapi-client = "pystapi_client.cli:cli"
+stapi-cli = "pystapi_client.scripts.cli:cli"
```

I wonder if we want to go "all the way" and just take the `stapi` name?

```suggestion
stapi = "pystapi_client.scripts.cli:cli"
```